### DISCUSSION TITLE
ipywidgets

### DISCUSSION THREAD

**Discussion by BryanArizmendi** [LaunchX-InnovaccionVirtual/CursoIntroPython#178] [Q&A]
Hola, estoy en ceros y empecé con la preparación del entorno de trabajo, he seguido las instrucciones, pero al instalar la biblioteca llamada ipywidgets no sale el siguiente paso que es reiniciar el kernel. Ya revise algunas dudas de otros compañeros y ninguna me ha funcionado alguien me puede ayudar? 

![ipywidgets](https://user-images.githubusercontent.com/99289651/153696706-f169dd57-41fd-4a0c-a6e9-83c551b16cb6.PNG)

**Comment by OscarPedroza** [comment]
Hola, si no me equivoco tu librería se instalo correctamente, lo de actualizar el kernel es para poder usar los paquetes sin problema.

**Comment by OscarPedroza** [comment]
¿Podrías mandar una captura del texto del mensaje restante para ver si no te indica algún problema?

**Reply by BryanArizmendi** [reply]
![ipywidgets 2](https://user-images.githubusercontent.com/99289651/153697174-49cd00c3-d0e7-4e14-bad0-4a418cb46707.PNG)
![ipywidgets3](https://user-images.githubusercontent.com/99289651/153697181-c569bc5e-caac-4282-a72f-4647f3668543.PNG)

**Reply by JosueVerAlar** [reply]
Todo está correcto, se instaló sin problema el paquete, solo revisa la parte superior derecha para seleccionar el kernel nuevamente. En caso de que no te funcione reinicia VS Code. Omite los Warning, solo te indican que podrías usar una versión más reciente pero puedes omitirlos sin problema, saludos!

**Reply by OscarPedroza** [reply]
Ok,  si la librería esta instalada, ya te deberia dejar trabajar sin problema.

**Reply by OscarPedroza** [reply]
PD: Asi puedes borrar los mensajes de outpust para que ya no te lo muestre.
![Captura2](https://user-images.githubusercontent.com/83786890/153697356-4ddf8550-a93c-42a6-9dd8-04ed3229bd60.PNG)

**Reply by BryanArizmendi** [reply]
Muchas gracias

**Comment by Ingrzs** [comment]
Sigue asi, actualizar el kernel es para cuando tienes mas una versión de Python instalada, como en este caso solo es has hecho una instalación de python ya en automático se te activa el kernel Actualizado

**Reply by omarfrias-maker** [reply]
¿Cómo se actualiza el kernell ?
